src/main/java/com/choelinska/horrorlibz/controllers/UserController.java

```diff
@@ -19,15 +19,24 @@
 
     private UserService userService;
 
-    private ModelMapper mapper;
-
     public UserController(UserService userService, ModelMapper mapper){
         this.userService =  userService;
-        this.mapper = mapper;
     }
 
     @PostMapping(consumes= MediaType.APPLICATION_JSON_VALUE)
     public void addUser(@RequestBody UserTo user){
         userService.saveUser(user);
     }
+
+    //Get all users sorted by username DESC
+    @GetMapping("/sortedNameDesc")
+    public List<UserEntity> getSorted(){
```

Comment:
Entities should only be used when interacting with the database. In the controller we want to return the TO, so please change the return type to UserTo.

Reply:
Of course I meant the return type to be the List<UserTo>